tls/s2n_connection_evp_digests.h

```diff
@@ -30,16 +30,7 @@ struct s2n_connection_prf_handles {
 
 struct s2n_connection_hash_handles {
     /* Handshake hash states */
-    struct s2n_hash_evp_digest md5;
```

Comment:
Why are we allowed to get rid of these digests?

Reply:
We're not really losing any digests here. These structures exist just to make temporary copies of hashes while we wipe s2n_connection, then restore those hashes later. Since the hashes are now a separate chunk of memory, they don't get zeroed when s2n_connection is zeroed and don't need to be saved/restored anymore.